models/_sector/lending/borrow/ethereum/_schema.yml

```diff
@@ -475,3 +475,46 @@ models:
       - *block_number
       - *tx_hash
       - *evt_index
+
+  - name: strike_ethereum_base_borrow
+    meta:
+      blockchain: ethereum
+      sector: lending
+      project: strike
+      contributors: tomfutago
+    config:
+      tags: ['lending', 'borrow', 'compound', 'strike', 'ethereum']
+    description: "Strike borrow transactions on Ethereum"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - transaction_type
+            - token_address
+            - tx_hash
+            - evt_index
+      - check_seed:
+          seed_file: ref('strike_ethereum_base_borrow_seed')
+          match_columns:
+            - transaction_type
+            - token_address
+            - tx_hash
+            - evt_index
+          check_columns:
+            - block_number
+            - borrower
+            - amount
```

Comment:
Maybe we can make a `check_base_borrow_seed` generic test.

Reply:
not a requirement, but good suggestion. won't leave that as a blocker. i believe `nft.trades` may be setup similar to here.